### PR TITLE
fix: Fix bugs in manifests

### PR DESCRIPTION
**Pull request by gaocegege**
The manifest still breaks.

Signed-off-by: Ce Gao <cegao@tensorchord.ai>

### CODE REVIEW

File: manifests/values.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ image:
   repository: tensorchord/envd-server
   pullPolicy: Always
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "0.0.9"
+  tag: "0.0.14"
```
Comment (VoVAllen) [inline manifests/values.yaml:11]: It's bumped to 0.0.16 now

Comment (gaocegege) [reply manifests/values.yaml:11]: Gotcha.

**Comment by gaocegege** [comment]
PTAL

/assign @kemingy @VoVAllen

**Review by VoVAllen — DISMISSED** [review]
lgtm

**Review by kemingy — APPROVED** [review]
/lgtm

**Pull request merged**